src/Sonarr.Http/ApiInfoController.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.AspNetCore.Mvc;
+using NzbDrone.Core.Configuration;
+
+namespace NzbDrone.Http
+{
+    public class ApiInfoController : Controller
+    {
+        private readonly IConfigFileProvider _configFileProvider;
+
+        public ApiInfoController(IConfigFileProvider configFileProvider)
+        {
+            _configFileProvider = configFileProvider;
+        }
+
+        [HttpGet("/api")]
+        [Produces("application/json")]
+        public object GetApiInfo()
+        {
+            return new
+            {
+                Latest = $"{_configFileProvider.UrlBase}/api/v3",
+                v2 = ApiStatus.Removed,
```

Comment:
To reduce schema changes, would it be better to do this instead?

```json
"removed": ["v2"],
"deprecated": [],
"current": "v2"
```

And technically, `removed` isn't even needed I guess. If it's not deprecated and not current, it's obviously removed.

Thoughts? I don't have a strong opinion one way or the other. Just food for thought.

Reply:
Personally I like this format better than what I suggested, both from the schema perspective and the fact that you can exclude the removed ones which makes it even cleaner.

Reply:
As long as we are fine eliminating the return of the full api root (which includes any urlbase that user has setup) and just returning the version we can switch to this.

Reply:
I think that's fine, they managed to hit this endpoint which also includes the URL base.

Reply:
> As long as we are fine eliminating the return of the full api root (which includes any urlbase that user has setup) and just returning the version we can switch to this.

I'm personally OK with that because at least in Recyclarr's use case, I ask the user to input their base URL. Most software I've seen that interacts with Sonarr also asks for this information from the user.

I think as long as the general structure of the URL never changes, that `v3` is sufficient to fill in the gap:

```
{baseurl}/api/{version}
```

- `{baseurl}`: Requested from the user (client responsibility)
- `{version}`: Nondeterministic; provided by API

Thanks again Q!